modules/library/main/src/main/java/org/geotools/data/ows/ControlledHttpClientFactory.java

```diff
@@ -0,0 +1,25 @@
+/*
+ *    GeoTools - The Open Source Java GIS Toolkit
+ *    http://geotools.org
+ *
+ *    (C) 2020, Open Source Geospatial Foundation (OSGeo)
+ *
+ *    This library is free software; you can redistribute it and/or
+ *    modify it under the terms of the GNU Lesser General Public
+ *    License as published by the Free Software Foundation;
+ *    version 2.1 of the License.
+ *
+ *    This library is distributed in the hope that it will be useful,
+ *    but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ *    Lesser General Public License for more details.
+ */
+package org.geotools.data.ows;
+
+/** @author ImranR */
+public class ControlledHttpClientFactory {

```

Comment:
The proposal uses different names:

![image](https://user-images.githubusercontent.com/325433/76219198-bf355480-6215-11ea-96e7-a81bf737c00c.png)

Please be consistent on both sides.

Reply:
understood, I will change on proposal